### DISCUSSION TITLE
Option to hide countdown timer on delayed page

### DISCUSSION THREAD

**Discussion by xuiqzy** [proginosko/LeechBlockNG#397] [Ideas]
Seeing the progress of the countdown counting down on the delayed page might make one invested in completing the waiting for the countdown instead of just closing the tab. It also makes the delayed page slightly less boring, which is not good if one wants to incentivize just closing the tab if the waiting is not worth it and too boring. Both these effects seem to be true for me at least.

Idea and detailed reasoning from [Dopanope extension from bossasaservice](https://bossasaservice.com/dopanope) (only chrome): 

> If we show the timer, it works counterintuitively. It seems to make you think "Just 22 seconds more" and make you more willing to wait. But waiting an unknown amount of time [see my related suggestion of randomizing delay a bit] causes you to think "ugh, this isn't worth it" and close the tab in disgust.
> 
> This is probably because of the [Endowed Progress Effect](https://www.jstor.org/stable/10.1086/500480), the psychological tendency for us to be more motivated to complete a task or reach a goal when we perceive we've made progress towards it. Seeing a countdown increases the sense of progress towards the end goal, making people more willing to wait. On the other hand, waiting an unspecified amount of time without any sense of progress leads to frustration. You close the tab, and end up with more time on hand, more work done, and less knowledge of the minutiae of your high school friends' life, which is the best outcome we can ask for.

Implementing [Idea: Custom delay page in Firefox >57?](https://bossasaservice.com/dopanope#faqs) would probably also solve this, although just this is probably an easier task.

A global option to disable it for every block set would be totally enough.
I would very much like to see that supported here in LB, which has many other useful features I use already and has a Firefox extension.

**Comment by proginosko** [comment]
See #518 for a way to hide the countdown timer using custom CSS.
